zerver/models.py

```diff
@@ -721,6 +725,9 @@ class Realm(models.Model):  # type: ignore[django-manager-missing] # django-stub
     # they will not be available regardless of users' personal settings.
     enable_read_receipts = models.BooleanField(default=False)
 
+    # Names for system group
+    ADMINISTRATORS_GROUP_NAME = "@role:administrators"
+
```

Comment:
I think it would be fine if we move all entries of system_groups names from `UserGroup` model to `Realm` model as a follow up to this PR.

Reply:
I think an even better plan might be to move them to a central object or dataclass, rather than hanging off a model at all, but agreed this is a good follow-up.